haiku-apps/beohms/beohms-0.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Ohm's Law calculator"
+DESCRIPTION="BeOhms is a calculator that is based of of Ohm's Law. This \
+calculator solves either voltage, resistance, or current, given two of the \
+variables."
+HOMEPAGE="https://github.com/HaikuArchives/BeOhms/"
+COPYRIGHT="1999 Kyle Crane"
+LICENSE="Public Domain"
+REVISION="1"
+commit="43cf57c95ae70a98c64e01c4d600eaa0ca7519ed"
+SOURCE_URI="https://github.com/HaikuArchives/BeOhms/archive/$commit.tar.gz"
+CHECKSUM_SHA256="65901b77d0deb15b0b327a38b0965382d171ff178f3737fadd4a3741aa7c9b14"
+SOURCE_FILENAME="BeOhms-$commit.tar.gz"
+SOURCE_DIR="BeOhms-$commit"
+PATCHES="beohms-0.1.patchset"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	beohm$secondaryArchSuffix = $portVersion
```

Comment:
Since your recipes builds fine on all primary architectures, you don't need to have support for secondary architectures.

This means you ~~can~~ should  remove all occurences of $secondaryArchSuffix and {secondaryArchSuffix}.